content/docs/operating/security.md

```diff
@@ -136,7 +139,7 @@ wherever it is stored.
 ## Denial of Service
 
 There are some mitigations in place for excess load or expensive queries.
-However, if too many or too expensive queries/metrics are provided components
+However, if too many or too expensive queries/metrics are provided, components
```

Comment:
This is fine as-is.

Reply:
I disagree.

Reply:
"However components will fall over" does not make sense.